src/app/api/file/[fileId]/route.schema.ts

```diff
@@ -0,0 +1,70 @@
+import { z } from "zod";
+import { unauthorizedErrorSchema, unknownErrorSchema } from "@api/route.schema";
+/*
+  id           String   @id @default(auto()) @map("_id") @db.ObjectId
+  date         DateTime // will restrict hours to midnight 
+  filetype     String
+  url          String
+  seniorId     String   @db.ObjectId
+  senior       Senior   @relation(fields: [seniorId], references: [id], onDelete: Cascade)
+  Tags         String[]
+
+  @@unique([seniorId, date])
+*/
+
+export const File = z.object({
```

Comment:
Since we defined the same object in `api/file/route.schema.ts`, can we refactor and put this schema in `server/model/index.ts`?